metashare/api/models.py

```diff
@@ -90,3 +94,24 @@ def valid_token_for(self):
         if all(self.token) and self.org_id:
             return self.org_id
         return None
+
+
+class Product(HashIdMixin):
+    name = models.CharField(max_length=50, unique=True)
+    repo_name = models.SlugField(unique=True)
+    version_number = models.CharField(max_length=50)
+    description = MarkdownField()
+    is_managed = models.BooleanField(default=False)
+    license = ArrayField(
+        models.CharField(max_length=64, choices=LICENSES), blank=True, default=list
+    )
+
+
+class ProductDependency(HashIdMixin):
+    url = models.URLField()
+    product = models.ForeignKey(
+        Product, on_delete=models.CASCADE, related_name="dependencies"
+    )
+
+    def __str__(self):
+        return self.url
```

Comment:
Dependencies should work like this:

Products should know their own github URL and should have a checkbox for whether the product is available to choose as a dependency when adding a new product.

Let me know if I'm forgetting a reason we need it, but I don't think we need/want the relationship between a product and its dependencies to be part of the MetaShare data model. The truth for this is the list of dependencies in cumulusci.yml, and I don't want us to have to worry about keeping this in sync with that unless there's a reason.

Reply:
OK; I began work on `product-list`, but drew from the descriptions in the `product-create` story; I understand now that a number of those things are not things we need to store in the DB, but rather take in as input for when we initialize a new product repo. Gotcha.